account_avatax/models/account_move.py

```diff
@@ -465,6 +489,9 @@ def _avatax_prepare_line(self, sign=1, doc_type=None):
         amount = sign * line._get_avatax_amount()
         if line.quantity < 0:
             amount = -amount
+        avatax_amt = 0.0
+        if line.move_id.move_type == "out_refund":
```

Comment:
Also consider `avatax_amt_line_override`?
```suggestion
        if line.avatax_amt_line_override and line.move_id.move_type == "out_refund":
```